vscode-react/src/components/deployments/organisms/executePopper.component.tsx

```diff
@@ -1,89 +1,199 @@
-import React, { MouseEvent } from "react";
+import React, { MouseEvent, useState, useEffect } from "react";
 
 import { translate } from "@i18n";
 import { Button } from "@react-components/atoms/button.component";
-import { SessionEntrypoint } from "@type/models";
-import { VSCodeDropdown } from "@vscode/webview-ui-toolkit/react";
+import { VSCodeDropdown, VSCodeTextField } from "@vscode/webview-ui-toolkit/react";
 
 interface ExecutePopperProps {
-	files: Record<string, SessionEntrypoint[]>;
-	functions: SessionEntrypoint[];
+	files?: string[];
+	functionName: string;
 	selectedFile: string;
-	selectedFunction: string;
 	onFileChange: (file: string) => void;
 	onFunctionChange: (func: string) => void;
-	onStartSession: (event?: MouseEvent<HTMLElement>) => void;
+	onStartSession: (params: { key: string; value: string }[]) => void;
 	onClose: () => void;
 	displayedErrors: Record<string, boolean>;
 }
 
-export const ExecutePopper: React.FC<ExecutePopperProps> = ({
+interface InputParameter {
+	key: string;
+	value: string;
+	keyError?: string;
+	valueError?: string;
+}
+
+export const ExecutePopper = ({
 	files,
-	functions,
+	functionName,
 	selectedFile,
-	selectedFunction,
 	onFileChange,
 	onFunctionChange,
 	onStartSession,
 	onClose,
 	displayedErrors,
-}) => {
+}: ExecutePopperProps) => {
+	const [inputParameters, setInputParameters] = useState<InputParameter[]>([]);
+	const [hasValidationErrors, setHasValidationErrors] = useState(false);
+
+	const validateParameters = () => {
+		const updatedParams = inputParameters.map((param) => ({
+			...param,
+			keyError: param.key.length === 0 ? "Key must not be empty" : undefined,
+			valueError: param.value.length === 0 ? "Value must not be empty" : undefined,
```

Comment:
```suggestion
			keyError: param.key.length === 0 ? translate().t("reactApp.deployments.inputParameters.errors.keyEmpty") : undefined,
			valueError: param.value.length === 0 ? translate().t("reactApp.deployments.inputParameters.errors.valueEmpty") : undefined,
```
and add to i18n folder

Reply:
done